public/stylesheets/palette.css

```diff
@@ -273,3 +273,7 @@ input.form-control-error[type="password"],
 .notice-banner {
   background-color: #1d70b8;
 }
+
+.text--right {
```

Comment:
Can you move this to our general-purpose VAT CSS file instead. `palette.css` is for changes regarding the colour palette.